releases/process.md

```diff
@@ -105,10 +105,11 @@ Generally the process for Code Freeze occurs around 10:00AM PT on the first day
 Project Leads
 {: .label .label-blue}
 
-1. Move any open pull requests targeting `branch-M.B` to target `branch-M.C` instead
-2. Wait for confirmation from operations on the branch switch
-3. Continue `M.C` development
-4. Respond promptly to operations if any issues are found with the `M.B` release
+1. Final call for new release artifacts in PIC synch (2 weeks prior to cuDF burndown)
```

Comment:
```suggestion
1. Inform operations team of any new release artifacts (packages, wheels, containers) no later than 2 weeks prior to cuDF burndown.
```

Reply:
Didnt want to mention PIC sync as these docs are public and PIC syncs are internal NV-only.